homeassistant/components/tradfri/base_class.py

```diff
@@ -47,14 +47,14 @@ def __init__(
     ) -> None:
         """Initialize a device."""
         self._api = handle_error(api)
-        self._device: Command | None = None
+        self._device: Command = device
```

Comment:
The `device` should be an instance of `Device`.

Reply:
Done (there is however a comment in the library, which I am going to change as it is misleading).